tck/src/main/java/ee/jakarta/tck/concurrent/api/ContextService/ContextServiceTests.java

```diff
@@ -42,6 +44,9 @@ public static WebArchive createDeployment() {
 		return ShrinkWrap.create(WebArchive.class)
 				.addPackages(true,  ContextServiceTests.class.getPackage());
 	}
+	
+    @Resource(lookup = TestConstants.DefaultContextService)
```

Comment:
Do we need to explicitly specify default values? I would use the `lookup` only when needed.

Reply:
In some cases, yes.  
This resource is injected by the Arquillian enricher and not the application server itself.